README-KO.md

```diff
@@ -1156,4 +1158,29 @@ class JavaFriendlyAPI {
 
 `URL`의 [동일성 검사](http://docs.oracle.com/javase/7/docs/api/java/net/URL.html#equals(java.lang.Object)) 는 사실 IP 주소를 알아내기 위해 (블로킹) 네트워크 호출을 합니다.  `URI` 클래스는 필드의 동일성을 확인하고 `URL`의 상위 집합 입니다.
 
+### <a name='misc_well_tested_method'>이미 존재 하는 함수를 다시 개발하는 것 보다는 기존의 잘 테스트 된 함수 사용</a>
+
+이미 존재하며 잘 테스트 되어있는 함수가 있고 이 함수가 어떤 성능 문제도 갖고 있지 않을 때에는, 이를 사용 하도록 합니다. 이러한 함수를 다시 구현하면 버그가 발생할 수 있으며, 이를 테스트하는데 시간이 필요합니다 (어쩌면 이 함수를 테스트 해야 한다는 것을 잊어버릴 수도 있습니다!).
+
+
+  ```scala
+  val beginNs = System.nanoTime()
+  // 시간 측정을 위한 일을 합니다.
+  Thread.sleep(1000)
+  val elaspedNs = System.nanoTime() - beginNs
+
+  // 아래와 같이 하지 않습니다. 아래는 매직 넘버를 사용하고 있어서 쉽게 실수 할 수 있습니다.
+  val elaspedMs = elaspedNs / 1000 / 1000
+
+  // 아래와 같이 Java의 TimeUnit API 를 사용합니다.
+  import java.util.concurrent.TimeUnit
+  val elaspedMs2 = TimeUnit.NANOSECONDS.toMillis(elaspedNs)
+
+  // 아래와 같이 Scala의 Duration API를 사용합니다.
+  import scala.concurrent.duration._
+  val elaspedMs3 = elaspedNs.nanos.toMillis
+  ```
 
+예외 경우:
+- 이미 존재하며 잘 테스트 되어있는 함수를 사용하려면 새로운 종속성을 추가 해야 합니다. 만약 이러한 함수가 상당히 간단하다면, 다시 구현하는 것이 새로운 종속성을 추가하는 것 보다 낫습니다. 하지만, 테스트를 해야 된다는 것을 잊지 말아야 합니다.
```

Comment:
The original one, "이미 존재하며 잘 테스트 되어있는 함수를 사용하려면 새로운 종속성을 추가 해야 합니다", means basically using the method means always adding a new dependency.

I think the suggestion, "이미 잘 테스트 되어있는 함수를 사용하기위해 새로운 종속성(dependency)을 추가해야 하는 경우" means, in case that using the method requires a new dependency.

I thought using the method always requires adding a new dependency (the codes using the method is dependent on the method).